packages/dd-trace/src/appsec/index.js

```diff
@@ -1,8 +1,76 @@
 'use strict'
 
-function enable () {
+const fs = require('fs')
+const path = require('path')
+const log = require('../log')
+const RuleManager = require('./rule_manager')
+const { INCOMING_HTTP_REQUEST_START } = require('../gateway/channels')
+const Gateway = require('../gateway/engine/index')
+const Addresses = require('./addresses')
+const Reporter = require('./reporter')
+
+function enable (config) {
+  try {
```

Comment:
This try is wrapped around a lot of stuff that probably shouldn't ever fail. Can it be scoped down a bit?